### PR TITLE
Enable links between articles via route changes

### LINKED ISSUE #514: Add relative links in data
**Is your feature request related to a problem? Please describe.**
We cannot currently test links through our application because there is no relative links.

**Describe the solution you'd like**
Add some relative links to our data.

**Additional context**
We should add relative links to production dataset, so we can properly test it with Cypress.

### PR DESCRIPTION
**Pull request by sareh**
Resolves #514

This PR updates the route for articles to be of this format `/news/articles/c[xxxxxxxxxx]o` where the `x`'s are alphanumeric values.

You can view two examples of articles with links referencing one another here:
http://localhost:7080/news/articles/c0000000025o
http://localhost:7080/news/articles/c0000000027o 

- [x] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ npm install
...
 To run this application locally, with hot-reloading, run: `npm run dev`.
 
-The application will start on [http://localhost:7080/article/scenario-id](http://localhost:7080/article/scenario-id).
+The application will start on [http://localhost:7080/news/articles/c0000000001o](http://localhost:7080/news/articles/c0000000001o).
 
-These is a single route, `/article/:id`, where `id` is the filename of the static Article data, for example `scenario-01`.
+These is a single route, `/news/articles/:id`, where `id` is the filename of the static Article data, for example `c0000000001o`.
 
-**NOTE:** the `id` parameter is currently redundant as the Article component is hard-coded to fetch data from [scenario-01](https://github.com/bbc/simorgh/blob/latest/data/scenario-01.json). Furthermore, the Article component does not do anything with that data.
+**NOTE:** the `id` parameter is currently redundant as the Article component is hard-coded to fetch data from [scenario 1: c0000000001o](https://github.com/bbc/simorgh/blob/latest/data/c0000000001o.json). Furthermore, the Article component does not do anything with that data.
```
Comment (jtart) [inline README.md]: This is wrong now, and should be removed.

**Review by jtart — CHANGES_REQUESTED** [review]

**Review by jtart — DISMISSED** [review]

File: data/prod/c0000000025o.json [hunk 2]
```diff
@@ -191,7 +191,7 @@
                 "blockId": "8eb06d3b-4c93-9685-f2d1-cba0a9c49966",
                 "type": "paragraph",
                 "model": {
-                  "text": "It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George's Chapel and St George's Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c000000000ro)."
+                  "text": "It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George's Chapel and St George's Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c0000000025o)."
```
Comment (BogdanDogaru) [inline data/prod/c0000000025o.json]: Good spot, but have you considered linking this to `/news/articles/c0000000027o` so it's consistent with the non-prod scenarios and link to a different article?

File: data/prod/c0000000027o.json [hunk 3]
```diff
@@ -190,7 +190,7 @@
                 "blockId": "8fe17c4d-7138-eb79-90e6-135476782d87",
                 "type": "paragraph",
                 "model": {
-                  "text": "/Meghan's own bouquet was [placed on the tomb of the unknown warrior](/news/articles/c000000000po) at Westminster Abbey, following royal tradition."
+                  "text": "/Meghan's own bouquet was [placed on the tomb of the unknown warrior](/news/articles/c0000000027o) at Westminster Abbey, following royal tradition."
```
Comment (BogdanDogaru) [inline data/prod/c0000000027o.json]: Same comment as above, but with this linking to `/news/articles/c0000000025o` instead.

**Review by BogdanDogaru — CHANGES_REQUESTED** [review]

**Review by pjlee11 — DISMISSED** [review]

File: data/prod/c0000000025o.json (hunk 2 above)
Comment (sareh) [reply data/prod/c0000000025o.json]: Thanks for spotting this typo. I will make the change.

**Review by BogdanDogaru — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**